src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,12 +8,66 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String EXAMPLE = "a/123, Clementi Ave 3, #12-34, 231534";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses has to be in this format: a/BLOCK, STREET, UNIT, POSTAL_CODE";
+    public static final String ADDRESS_VALIDATION_REGEX = "(.+),(.+),(.+),(.+)";
+    public static final int ADDRESS_SIZE = 4;
 
-    public final String value;
+    //public final String value;
```

Comment:
remove unnecessary code completely; you can always recover it from previous versions RCS \#FTW